UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Great idea to add SwiftLint, it's really useful. Didn't get any errors / warnings from it just by running tests. However, building the project (CMD + B) is reporting all the warnings (trailing_newline) in .swift.out for fonts, colors... but no errors for the force_cast. I'm also getting some `no such files or directory` from Xcode.

I'm not update from the master, that may be an explanation why I'm getting all these warnings, but don't know why I don't have the force_cast error. I will give a try later.